src/app/ui/upload/upload.component.html

```diff
@@ -11,16 +10,21 @@ <h5 class="mb-0">
       </div>
       <div id="collapseOne" class="collapse show" aria-labelledby="headingOne" data-parent="#accordionExample">
         <div class="card-body">
-          <div class="input-group mb-3">
-            <div class="custom-file">
-              <input type="file" class="custom-file-input" id="inputGroupFile01"  accept="application/pdf" #file (change)="onFileSelected($event)" aria-describedby="inputGroupFileAddon01">
-              <label class="custom-file-label" for="inputGroupFile01">Choose file</label>
+          <div class="box has-advanced-upload" [ngClass]="{'is-dragover':isDragged == true}" (drop)="onDrop($event)" (dragover)="onDragOver($event)"
+               (dragleave)="onDragLeave($event)" enctype="multipart/form-data">
+            <div class="box-input">
+              <i class="fa fa-upload box-icon"></i>
```

Comment:
We imported a FontAwesome component. Use the `<fa>` tag, instead of the HTML default i tag.